src/modelbench/benchmark_runner.py

```diff
@@ -503,13 +473,37 @@ def _check_ready_to_run(self):
     def run(self) -> BenchmarkRun:
         self._check_ready_to_run()
         benchmark_run = BenchmarkRun(self)
+        benchmark_run.journal.raw_entry(
```

Comment:
Maybe we should make it clear which tests are associated with which benchmarks? Also, I think we may want to include more information about the SUTs/tests at some point. e.g. class type, plugin, initialization records, test-dependencies/file names

Reply:
This stuff can probably wait for another PR though.

Reply:
I think that's a fine follow-on PR. You should feel free.